configure.in

```diff
@@ -2,7 +2,7 @@ dnl vim:ts=2:sw=2:et:
 AC_INIT(jlog.c)
 AC_CONFIG_HEADER(jlog_config.h)
 
-AC_SUBST(CPPFLAGS, "$CPPFLAGS -I/usr/java/include -I/usr/java/include/solaris")
+AC_SUBST(CPPFLAGS, "$CPPFLAGS -I/usr/java/include -I/usr/java/include/solaris -I/opt/circonus/jdk1.8.0_74/include -I/opt/circonus/jdk1.8.0_74/include/solaris")
```

Comment:
/opt/circonus?  This doesn't belong here... we should be building with CPPFLAGS="-I..." environments to accommodate these requirements.